spec/controllers/ops_controller/settings/automate_schedules_spec.rb

```diff
@@ -77,6 +77,17 @@
         ops.fetch_target_ids
       end
     end
+
+    context 'fetching target ids' do
+      ['MiqGroup', 'User', 'Tenant'].each do |klass|
+        it "fetches appropriate ids for #{klass}" do
```

Comment:
Just a small semantic nitpick, have the iteration outside the context, i.e. separate context for each `klass` and so the `it` doesn't have to be dynamic.